packages/atomic/cypress/integration/facets/date-facet/date-facet.cypress.ts

```diff
@@ -53,6 +53,7 @@ describe('Date Facet Test Suites', () => {
         cy.wait(RouteAlias.search);
         cy.wait(RouteAlias.analytics);
         selectFacetValueAt(0, dateField, dateFacetComponent);
+        cy.wait(RouteAlias.search);
```

Comment:
Maybe this was causing the following block to catch the previous request.
I couldn't reproduce the flakiness locally anyways, only seen on jenkins